lib/src/main/java/growthbook/sdk/java/GBFeaturesRepository.java

```diff
@@ -382,7 +382,13 @@ private void onSuccess(Response response) throws FeatureFetchException {
             ResponseBody responseBody = response.body();
             if (responseBody == null) {
                 throw new FeatureFetchException(
-                    FeatureFetchException.FeatureFetchErrorCode.NO_RESPONSE_ERROR
+                        FeatureFetchException.FeatureFetchErrorCode.NO_RESPONSE_ERROR
+                );
+            }
+            if (response.code() != 200) {
+                throw new FeatureFetchException(
```

Comment:
Seems like if user of the SDK doesn't catch `FeatureFetchException` exception in runtime will appear.